ssh_certificate_parser/__init__.py

```diff
@@ -91,6 +109,11 @@ def asdict(self):
         dct['cert_type'] = dct['cert_type'].name
         dct['signature'] = base64.b64encode(dct['signature']).decode('ascii')
         dct['ca_fingerprint'] = self.ca.fingerprint
+        dct['pubkey_parts'] = dict(
+            (k, base64.b64encode(v).decode('ascii'))
```

Comment:
this is not the prettiest indentation i've ever seen.  i dunno what the standard says it should be though.